sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -0,0 +1,25 @@
+package com.vimeo.dummy.sample_kotlin
+
+import com.google.gson.annotations.SerializedName
+import com.vimeo.stag.UseStag
+
+/**
+ * Test kotlin model
+ *
+ * Created by restainoa on 5/8/17.
+ */
+@UseStag
+class KotlinSamples {
+    var stringField: String? = null
+
+    var nonNullStringField: String = "default"
+
+    var intField: Int? = null
+
+    var longField: Long = 1
+
+    @SerializedName("boolean_field")
+    var booleanField: Boolean? = null
+
+    var testField: Any? = null
```

Comment:
I tested out using `internal` and it's not viable to support until stag is generating Kotlin output for kotlin modules instead of java. I tested by modifying the `myField` field and making only the setter `internal`. See the below code for the temporary java code generated by the kotlin plugin. Note the different naming for the setter that contains the module name in it... something that we don't have access to within the context of an annotation processor:

```java
@com.vimeo.stag.UseStag()
public final class KotlinSamples {
    @org.jetbrains.annotations.Nullable()
    private java.lang.String stringField;
    @org.jetbrains.annotations.Nullable()
    private java.lang.Object testField;
    
    @org.jetbrains.annotations.Nullable()
    public final java.lang.String getStringField() {
        return null;
    }
    
    public final void setStringField(@org.jetbrains.annotations.Nullable()
    java.lang.String p0) {
    }
    
    @org.jetbrains.annotations.Nullable()
    public final java.lang.Object getTestField() {
        return null;
    }
    
    public final void setTestField$sample_kotlin_model_debug(@org.jetbrains.annotations.Nullable()
    java.lang.Object p0) {
    }
    
    public KotlinSamples() {
        super();
    }
}
```